karma.conf.js

```diff
@@ -22,6 +22,8 @@ module.exports = function(config) {
               if (request.url === '/base/data' && request.method === 'POST') {
                 var body = '';
 
+                response.setHeader('Access-Control-Allow-Origin', '*');
```

Comment:
In our testing, we found `*` doesn't work with `useCredentials`, and you have to pretty much echo back the origin provided in the header.